jdbc/composite-ids/src/main/java/example/springdata/jdbc/compositeid/EmployeeRepository.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2025 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package example.springdata.jdbc.compositeid;
+
+import org.springframework.data.repository.ListCrudRepository;
+
+
+/**
+ * Repositories for {@link Employee} instances.
+ *
+ * @author Jens Schauder
+ * @see InsertRepository
+ * @see InsertRepositoryImpl
+ */
+interface EmployeeRepository extends ListCrudRepository<Employee, EmployeeId>, InsertRepository<Employee> {
```

Comment:
Using fewer dependencies (i.e. just `Repository`) would be a welcome simplification as that is what we advocate for.

Reply:
Done.